rdr_service/model/bq_base.py

```diff
@@ -567,6 +568,9 @@ def update(dest, src, schema):
             :return: dict
             """
             for key, val in src.items():
+                if not isinstance(key, str):
+                    logging.warning('Dict key is not a string.')
```

Comment:
How noisy will this be based on what you saw on stable?  Or, you don't expect it to really happen on prod so no risk of high volume of warnings?